client/src/stockfish_interface_desktop.rs

```diff
@@ -0,0 +1,41 @@
+use crate::arrows::Arrows;
+use crate::stockfish_client::{process_output, MOVES};
+use async_process::{Child, Command, Stdio};
+use async_std::io::BufReader;
+use async_std::prelude::*;
+use dioxus::prelude::*;
+use futures::executor::block_on;
+use std::io::Result;
+
+pub type Process = Child;
```

Comment:
How come we didn't keep it as `Child` ?

Reply:
So in the interface for web, it's a `js_sys::Object`. It just sort of abstracts the interface so `stockfish_client` can just refer to `Process`.